LongestName/script.js

```diff
@@ -0,0 +1,25 @@
+let name1 = "Chris";
+let name2 = "Jacob";
+let name3 = "Janok";
+
+let length1 = name1.length;
+let length2 = name2.length;
+let length3 = name3.length;
+
+
+if (length1 > length2 && length1 > length3) {
```

Comment:
When writing a complex conditional, you can use more parentheses to group together related logic:

```suggestion
if ((name1.length > name2.length) && (name1.length > name3.length) {
```

Reply:
See more details in my slack thread about conditionals